helmfile/charts/networkpolicy-service/templates/fluentd-np.yaml

```diff
@@ -0,0 +1,125 @@
+{{ if .Values.fluentd.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: fluentd-default-deny-all
+  namespace: fluentd
+spec:
+  podSelector: {}
+  policyTypes:
+    - Ingress
+    - Egress
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-fluentd-aggregator
+  namespace: fluentd
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: aggregator
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              app.kubernetes.io/component: forwarder
+              app.kubernetes.io/name: fluentd
+      ports:
+        - port: 24224
+    - from:
+        - podSelector:
+            matchLabels:
+              app.kubernetes.io/name: prometheus
+              app.kubernetes.io/instance: kube-prometheus-stack-prometheus
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring
+      ports:
+        - port: 24231
+  egress:
+    - to:
+{{- range $ip := .Values.global.objectStorage.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+{{- end }}
```

Comment:
Can we keep the go templates on the same indentation level as the yaml it ranges over?